### DISCUSSION TITLE
post/serverless-media-processing-with-aci

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#71] [Announcements]
# post/serverless-media-processing-with-aci

Mark Heath's Development Blog

https://markheath.net/post/serverless-media-processing-with-aci

**Comment by IntvPrime** [comment]
This is great. Looking to experiment with running ffmpeg in ACI, taking RTMP stream and outputting HLS (the way that AMS did with Live Events but they are being deprecated). Have you seen a successful way to do that?

**Reply by markheath** [reply]
I haven't tried that I'm afraid, but let us know how you get on with your experiment
